react_main/src/Main.jsx

```diff
@@ -95,6 +95,14 @@ function Main() {
                     ), "basic", true);
                 }
 
+                if (res.data.bdayChanged == false) {

```

Comment:
This makes it pop up every single time you load the website if you don't set your birthday?

Reply:
Yeah, I guess that could be taken out. True.